### ISSUE TITLE
It should be possible to set a timeout when using the client library on AppEngine

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Dragon-Born/google-api-python-client#9]
```
What steps will reproduce the problem?
1. Perform an HTTP call using the client library against a resource that takes 
longer than the standard App Engine timeout limit (5 seconds)
2. See DownloadError: ApplicationError: 5

What is the expected output? What do you see instead?
You should be able to set a timeout value for the client library and that 
should get propagated down to AppEngine's urlfetch so that a call which times 
out at a point that's greater than 5 seconds but less than the new timeout 
value will succeed

Please use labels and text to provide additional information.
appengine http
```

Original issue reported on code.google.com by `a...@google.com` on 9 Feb 2011 at 10:30

**Comment by GoogleCodeExporter** [comment]
```
http://codereview.appspot.com/4517044
```

Original comment by `jcgregorio@google.com` on 9 May 2011 at 1:55
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Fixed with sync to version 0.7.0 of httplib2.

http://code.google.com/p/google-api-python-client/source/detail?r=3ce2ae291c18f2
2704e53783150d9b4621a6d055
```

Original comment by `jcgregorio@google.com` on 14 Jun 2011 at 6:33
- Changed state: **Fixed**
